admin_guide/high_availability.adoc

```diff
@@ -523,15 +518,15 @@ ifdef::openshift-origin[]
 ----
 $ oadm ipfailover ipf-ha-router-us-west \
     --replicas=5 --watch-port=80 \
-    --selector="ha-router=geo-us-west" \
+    --selector="ha-svc-nodes=geo-us-west" \
     --virtual-ips="10.245.2.101-105" \
     --credentials="$KUBECONFIG" \
     --service-account=ipfailover --create
 ----
 ====
 endif::[]
 
-. Edit the deployment configuration:
+. Edit the deployment config:
```

Comment:
I think "configuration" is more accurate here. We should try to keep the full word around.